integr8ly/integreatly-operator
# Mgdapi 3421 release v0.36.0

# Issue link

https://issues.redhat.com/browse/MGDAPI-3421

# What

CRO version bump

# Verification steps
- deploy CCS cluster confirm that redis and posgres and blobstorage are provisioned 
quay.io/austincunningham/managed-api-service-index:1.17.0 
quay.io/austincunningham/managed-api-service-index:1.18.0
